backend/src/v5/models/users.js

```diff
@@ -20,7 +20,9 @@ const config = require('../utils/config');
 const db = require('../handler/db');
 const { events } = require('../services/eventsManager/eventsManager.constants');
 const { generateHashString } = require('../utils/helper/strings');
+const { logger } = require('../utils/logger');
 const { publish } = require('../services/eventsManager/eventsManager');
+const { sendResetPasswordEmail } = require('../../v4/mailer/mailer');
```

Comment:
should be using v5 mailer (even if it's just importing v4 mailer)

There should be no reference to v4 from v5 (apart from the ones that are importing the whole file e..g mailer, db)